modules/reindex/src/main/java/org/opensearch/index/reindex/TransportUpdateByQueryAction.java

```diff
@@ -105,7 +106,7 @@ protected void doExecute(Task task, UpdateByQueryRequest request, ActionListener
                 clusterState.nodes().getMinNodeVersion().onOrAfter(LegacyESVersion.V_6_7_0) == false,
                 // all nodes support sequence number powered optimistic concurrency control and we can use it
                 clusterState.nodes().getMinNodeVersion().onOrAfter(LegacyESVersion.V_6_7_0),
-                logger, client, threadPool, request, listener, scriptService, null);
+                logger, client, threadPool, request, listener, scriptService, null, Optional.empty());
```

Comment:
I thought you will create wrapper constructor in AbstractAsyncBulkByScrollAction.java, so that other sub classes are untouched if they don't need the interceptor.

Reply:
Yeah. Sorry for the misunderstanding. Will update the PR.

Reply:
Created a wrapper constructor